examples/simple-app/src/main.ts

```diff
@@ -125,30 +126,97 @@ client.on("peerJoined", (peer) => {
   remotePeers.appendChild(clone);
   toastInfo(`New peer joined`);
 });
+
 client.on("peerUpdated", (_peer) => {});
+
 client.on("peerLeft", (peer) => {
   const peerComponent = document.querySelector(`div[data-peer-id="${peer.id}"`)!;
   peerComponent.remove();
   toastInfo(`Peer left`);
 });
+
+const setUpSimulcastCheckbox = (element: HTMLDivElement, trackId: string, encoding: "l" | "m" | "h") => {
+  const simulcastInputL: HTMLInputElement | null = element.querySelector<HTMLInputElement>(
+    `.simulcast-input-radio-${encoding}`,
+  );
+  if (!simulcastInputL) return;
+
+  simulcastInputL.setAttribute("name", `${trackId}-simulcast`);
+
+  simulcastInputL.addEventListener("click", () => {
+    if (client.getRemoteTracks()[trackId]?.simulcastConfig?.enabled) {
+      client.setTargetTrackEncoding(trackId, encoding);
+    } else {
+      console.warn("You cannot set 'targetTrackEncoding' on a track that doesn't have an active simulcast.");
+      toastInfo("You cannot set 'targetTrackEncoding' on a track that doesn't have an active simulcast.");
+    }
+  });
+};
+
+const ENCODINGS: TrackEncoding[] = ["l", "m", "h"];
+
 client.on("trackReady", (ctx) => {
   console.log("On track ready");
+  console.log({ name: "trackReady", ctx });
+  if (!ctx.trackId) return;
+
   const peerId = ctx.endpoint.id;
   const peerComponent = document.querySelector(`div[data-peer-id="${peerId}"`)!;
-  const videoPlayer: HTMLVideoElement = peerComponent.querySelector(".remote-peer-template-video")!;
+
+  const videoPlayerTemplate = document.querySelector("#remote-peer-template-video")!;
+
+  // @ts-ignore
+  const videoWrapper = videoPlayerTemplate.content.cloneNode(true);
+
+  const tracksContainer: HTMLDivElement = videoWrapper.querySelector<HTMLDivElement>(`.remote-track-container`);
+  tracksContainer.dataset.trackId = ctx.trackId;
+
+  const videoPlayer: HTMLVideoElement = videoWrapper.querySelector<HTMLVideoElement>(`video`);
+
+  const container = peerComponent.querySelector(".remote-videos");
+
+  if (!container) throw new Error("Remote videos container not found!");
+
+  const simulcastContainer: HTMLDivElement = videoWrapper.querySelector<HTMLDivElement>(`.simulcast-enabled`);
+  simulcastContainer.innerHTML = (ctx?.simulcastConfig?.enabled || false).toString();
```

Comment:
```suggestion
  simulcastContainer.innerHTML = (ctx?.simulcastConfig?.enabled ?? false).toString();
```
I think that it is a much cleaner in this case, as we are dealing with a boolean value.